Description: Point javacc configuration at the jar files
 provided as part of the distribution.
Author: James Page <james.page@ubuntu.com>
Forwarded: not-needed

Index: lucene-solr/lucene/common-build.xml
===================================================================
--- lucene-solr.orig/lucene/common-build.xml	2012-05-23 10:00:18.885978436 +0100
+++ lucene-solr/lucene/common-build.xml	2012-05-23 10:01:28.666204839 +0100
@@ -138,7 +138,7 @@
   <property name="m2.repository.url" value="file://${maven.dist.dir}"/>
   <property name="m2.repository.private.key" value="${user.home}/.ssh/id_dsa"/>
 
-  <property name="javacc.home" location="${common.dir}"/>
+  <property name="javacc.home" location="/usr/share/java"/>
   <property name="jflex.home" location="${common.dir}"/>
 
   <path id="jflex.classpath">
@@ -152,7 +152,7 @@
 
   <path id="javacc.classpath">
     <fileset dir="${javacc.home}/">
-      <include name="bin/lib/*.jar"/>
+      <include name="javacc.jar"/>
     </fileset>
   </path>
 
